src/internal/test/data-testid-helpers.ts

```diff
@@ -0,0 +1,3 @@
+export const queryByTestId = <T extends Element>(container: HTMLElement, testId: string): T | null => {
```

Comment:
Just curious about this because you mentioned having interest in working on more tests later on — is this something we're going to be using in many tests? If so, a JSDoc comment to help future devs understand what it does would be great! 😄

Reply:
At least I tend to use that a lot and have come to like it ;) . So I added a JSDoc comment (also for the other extracted helper functions)